diff --git a/Lynx.Mobile/Lynx.CoreStandard/UserFacingException.cs b/Lynx.Mobile/Lynx.CoreStandard/UserFacingException.cs
new file mode 100644
index 0000000..34cb15a
--- /dev/null
+++ b/Lynx.Mobile/Lynx.CoreStandard/UserFacingException.cs
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace Lynx.Core
+{
+    public abstract class UserFacingException : Exception
+    {
+        protected UserFacingException()
+        {
+        }
+
+        protected UserFacingException(string message) : base(message)
+        {
+        }
+
+        protected UserFacingException(string message, Exception innerException) : base(message, innerException)
+        {
+        }
+    }
+}
